cpp/src/barretenberg/stdlib/primitives/bool/bool.test.cpp

```diff
@@ -50,8 +50,7 @@ TYPED_TEST(BoolTest, TestBasicOperations)
     bool result = composer.check_circuit();
     EXPECT_EQ(result, true);
 
-    auto gates_after = composer.get_num_gates();
-    EXPECT_EQ(gates_after - gates_before, 6UL);
```

Comment:
These tests that function like benchmarks should break, since we don't track gates, but they aren't testing circuit logic, so I felt comfortable deleting them.